packages/docs/src/content/03-components/form-controls/radio-group/overview.mdx

```diff
@@ -1,37 +1,77 @@
-# Beispiel
+# Playground
 
-<LiveCodeEditor />
+Verwende `<RadioGroup />` zusammen mit mehreren `<Radio />`, um eine RadioGroup
+darzustellen. Über der RadioGroup kann ein beschreibendes
+[Label](/03-components/content/label) verwendet werden. Es kann weggelassen
+werden, wenn die RadioGroup ohne verstanden wird oder wenn ein anderes
+beschreibendes Element – wie eine [Heading](/03-components/content/heading) – in
```

Comment:
Hier sollte noch erwähnt werden dass die Radio Group dann aber ein `aria-label` benötigt

Reply:
Stimmt, sollte erwähnt werden. In den Guidelines ist es unter dem Punkt Accessibility zu finden, kann aber da schnell übersehen werden. Hinter dem Satz im Overview kann eingefügt werden: "Wenn kein Label vorhanden ist, muss ein aria-label gesetzt werden."